fixposition_driver_lib/src/fixposition_driver.cpp

```diff
@@ -26,6 +26,19 @@
 #include <fixposition_driver_lib/helper.hpp>
 #include <fixposition_driver_lib/parser.hpp>
 
+#ifndef B460800
```

Comment:
Hi @floriansegginger, thanks for the MR. Can you explain what's going on with these defs?

Reply:
See my comment in https://github.com/fixposition/fixposition_driver/issues/29
These baud rates aren't part of the default ones provided on MacOS and they make compilation fail.